src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterTests.cs

```diff
@@ -2570,7 +2570,7 @@ class C
                 expectedIndentation: 8);
         }
 
-        [WpfFact, Trait(Traits.Feature, Traits.Features.SmartIndent)]
+        [WpfFact(Skip = "test is no longer correct due to language changes"), Trait(Traits.Feature, Traits.Features.SmartIndent)]
```

Comment:
Tracking what? Language changes? The deletion of this test?

---
In reply to: [91584597](https://github.com/dotnet/roslyn/pull/15548#discussion_r91584597) [](ancestors = 91584597)